<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/pics/plume-creator.png</normaloff>:/pics/plume-creator.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Change your language :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="langComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Change Plume style :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="styleComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Change your display :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="displayModeComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Save project every :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="autosaveTimeSpinBox">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string comment="seconds"> s</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>3600</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkUpdateAtStartupCheckBox">
           <property name="text">
            <string>Check update at startup</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="preventDoubleSpaceCheckBox">
           <property name="text">
            <string>Prevent multiple space characters between words</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="oneTabOnlyCheckBox">
           <property name="text">
            <string>Keep only one tab opened</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="noTabCheckBox">
           <property name="text">
            <string>Hide tabs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="textTab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Main Text Area</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Text Area Width :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="widthSpin">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Fullscreen Text Area</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="showFullScrollbarBox">
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Synopsis Area</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Default Text Font :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QFontComboBox" name="synFontCombo">
            <property name="currentFont">
             <font>
              <family>Liberation Serif</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="synSpin">
            <property name="minimum">
             <number>6</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="synShowScrollbarBox">
            <property name="text">
             <string>Show scrollbar</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Paragraph Indent :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="synIndentSpin">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Margin :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="synMarginSpin">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Note Area</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Default Text Font :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QFontComboBox" name="noteFontCombo">
            <property name="currentFont">
             <font>
              <family>Liberation Serif</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="noteSpin">
            <property name="minimum">
             <number>6</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="noteShowScrollbarBox">
            <property name="text">
             <string>Show scrollbar</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Paragraph Indent :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="noteIndentSpin">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Margin :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="noteMarginSpin">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stylesTab">
      <attribute name="title">
       <string>Styles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="styleListWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="renameStyleButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="pics.qrc">
               <normaloff>:/pics/edit-rename-32x32.png</normaloff>:/pics/edit-rename-32x32.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="removeStyleButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="pics.qrc">
               <normaloff>:/pics/list-remove-32x32.png</normaloff>:/pics/list-remove-32x32.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="addStyleButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="pics.qrc">
               <normaloff>:/pics/list-add-32x32.png</normaloff>:/pics/list-add-32x32.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Fixed Options :</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Text Font :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QFontComboBox" name="styleFontCombo">
              <property name="currentFont">
               <font>
                <family>Liberation Serif</family>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="styleSizeSpin">
              <property name="minimum">
               <number>6</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Paragraph Indent :</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="styleLeftMarginSpin">
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Spacing :</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="styleBottomMarginSpin">
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Text Size :</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Text Alignment :</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QComboBox" name="styleAlignComboBox">
              <item>
               <property name="text">
                <string>Left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Center</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Right</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Justified</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>1st Line Indent :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="styleFirstLineIndentSpin">
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="defaultStyleCheckBox">
              <property name="text">
               <string>Default Style</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Default Options :</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>These options can be modified with the right-click menu.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="styleItalicCheckBox">
              <property name="text">
               <string>Italic</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="styleBoldCheckBox">
              <property name="text">
               <string>Bold</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="styleUnderlineCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Underline</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="styleStrikeCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Strike Out</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noTabCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>oneTabOnlyCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noTabCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>oneTabOnlyCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
